CreeDictionary/API/models.py

```diff
@@ -505,6 +507,22 @@ def search(user_query: str) -> SortedSet[SearchResult]:
         return results
 
 
+def sort_by_user_query(user_query: str) -> Callable[[Any], Any]:
+    """
+    Returns a key function that sorts search results ranked by their distance
+    to the user query.
+    """
+    # mypy doesn't really know how to handle partial(), so we tell it the
+    # correct type with cast()
+    # See: https://github.com/python/mypy/issues/1484
+    return cmp_to_key(
+        cast(
```

Comment:
wow cast is jedi

Reply:
![cast](https://user-images.githubusercontent.com/44753941/73898193-cbf51000-4845-11ea-8d80-2a2faeb04d0e.jpg)